docs/SystemConfiguration.md

```diff
@@ -1,30 +1,30 @@
-# Default System Configuration
-
-The following versions of Hashistack components are used for creating the system:
-
-- **consul-template: 0.25.1**
-- **consul: 1.8.1**
-- **vault: 1.5.0**
-- **nomad: 0.12.1**
-
-The file holding the current version can be looked up [here](../ansible/group_vars/all/variables.yml)
-
-**Nomad policies** added can be looked up in the following location: [nomad policies](../ansible/templates/nomad-policies/)
-
-**Consul policies** added can be looked up in the following location: [consul policies](../ansible/templates/consul-policies/)
-
-By default Consul Acl policies are enabled on the system and the default policy is set to allow.
-
-## How to override the default System Configuration
-
-When the vagrant box is provisioned, it reads the data from the following environment file [.env_default](../template/.env_default) in order to set
-up the system. If you wish to override any of the default values then you can do so by adding that variable name and value in [.env](../template/.env) file. The property values in the .env file override the property values present in the .env_default file and thus makes it super easy to provision systems that suffice the relevant development needs.
-
-
-For example, in order to override the **consul acl default policy** from **allow** to **deny**, the following needs to be added to the **.env** file:
-
-
-**consul_acl_default_policy=deny**
-
-
+# Default System Configuration
+
+The following versions of Hashistack components are used for creating the system:
+
+- **consul-template: 0.25.1**
+- **consul: 1.8.1**
+- **vault: 1.5.0**
+- **nomad: 0.12.1**
+
+The file holding the current version can be looked up [here](../ansible/group_vars/all/variables.yml)
+
+**Nomad policies** added can be looked up in the following location: [nomad policies](../ansible/templates/nomad-policies)
+
+**Consul policies** added can be looked up in the following location: [consul policies](../ansible/templates/consul-policies)
+
+By default Consul Acl policies are enabled on the system and the default policy is set to allow.
+
+## How to override the default System Configuration
```

Comment:
There are two ways to do it:
1. Use env to toggle prebuild configuration on/off
2. Override of config files - user must provide; example  vagrant/conf/hashistack/consul/99-override.hcl

I would suggest to add 2nd approach to doc also

Reply:
You actually described it here 
- `template/vagrant/conf/consul/README.md `
- `template/vagrant/conf/nomad/README.md`
- `template/vagrant/conf/vault/README.md`

might be useful to have links from `docs/SystemConfiguration.md` to `template/vagrant/conf/consul/README.md `

Reply:
I agree. Please make a suggestion. I think we are still having a bit of overlap. Some documentation is only complete by reading about it in multiple places. ( this might be ok, but I think we should make sure not to duplicate to much )